tests/pthread/hello_thread.c

```diff
@@ -21,5 +22,5 @@ int main()
 {
 	pthread_t thread;
 	pthread_create(&thread, NULL, thread_main, NULL);
-	EM_ASM(noExitRuntime=true);
+	emscripten_unwind_to_js_event_loop();
```

Comment:
this seems good if we still have other tests that check the way that was removed? lgtm either if they exist, or with testing both here.

Reply:
Yeah, there's at least two such places.